assets/scss/_download.scss

```diff
@@ -0,0 +1,26 @@
+.download {
+    position: relative;
+}
+
+.download .button {
+    background-color: $pink;
+    
+}
+.download__title {
+    display: inline-block;
+}
+
+.download__title:after {
+    display: block;
+    content: "";
+    border-bottom: 3px solid $pink;
```

Comment:
To avoid code duplicating, I suggest moving this and the same code from _description.scss to the file _base.scss.
```css
.titles {
&::after {
      content: "";
      display: block;
      border-bottom: 3px solid $pink;
}
```